org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -42,47 +42,33 @@ $button-border-radius: 2px;
   .crm-form-block {
     &.crm-search-form-block > .crm-submit-buttons,
     &.crm-form-block > .crm-submit-buttons {
-      padding-top: $toolbar-top-space;
-      background: $crm-accordion-header-color !important;
-
-      %btn:not(.crm-button_qf_HRVacancy_next):not(.crm-button_qf_HRVacancy_cancel):not(#_qf_Application_upload) {
```

Comment:
It affected the vacancies page, I fix it on https://github.com/civicrm/civihr/pull/1138/commits/07d4b43b7c71b72d55b4de5118fe6c4ab18be073